avroProducer.go

```diff
@@ -27,7 +33,7 @@ func NewAvroProducer(kafkaServers []string, schemaRegistryServers []string) (*Av
 
 //GetSchemaId get schema id from schema-registry service
 func (ap *AvroProducer) GetSchemaId(topic string, avroCodec *goavro.Codec) (int, error) {
-	schemaId, err := ap.schemaRegistryClient.CreateSubject(topic, avroCodec)
+	schemaId, err := ap.schemaRegistryClient.CreateSubject(topic+"-value", avroCodec)
```

Comment:
oops sorry @cobolbaby 
but why do we need `-value`?

Reply:
Fix Error: io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException: Subject not found.; error code: 40401

Reply:
Ref: https://stackoverflow.com/questions/53731566/kafka-connect-jdbcsinkconnector-getting-error-retrieving-avro-schema-for-id

Reply:
ah thanks

Reply:
@cobolbaby can you please help to fix the wrong unittest?
```
--- FAIL: TestAvroProducer_Add (0.00s)
    /Users/user/works/dangkaka/go-kafka-avro/avroProducer_test.go:18: Error adding msg: unexpected end of JSON input
    /Users/user/works/dangkaka/go-kafka-avro/sync_producer.go:115: Expected to exhaust all expectations, but 1 are left.
FAIL
FAIL	github.com/dangkaka/go-kafka-avro	0.013s
Error: Tests failed.
```
a mismatch between the topic name with `-test`